blocks/init/src/Blocks/components/image/components/image-options.js

```diff
@@ -1,100 +1,139 @@
 import { checkAttr } from '@eightshift/frontend-libs/scripts/helpers';
+import { Responsive } from '@eightshift/frontend-libs/scripts/components';
 import manifest from './../manifest.json';
 
 export const ImageOptions = (attributes) => {
-	const { title } = manifest;
+	const {
+		title: manifestTitle,
+		breakpoints: manifestBreakpoints,
+		componentName: manifestComponentName,
+	} = manifest;
 
 	const {
 		setAttributes,
-		componentName = manifest.componentName,
-		label = title,
+		componentName = manifestComponentName,
+		label = manifestTitle,
 		imageShowControls = true,
 
 		imageUse = checkAttr('imageUse', attributes, manifest, componentName),
 
-		imageUrl = checkAttr('imageUrl', attributes, manifest, componentName),
 		imageAlt = checkAttr('imageAlt', attributes, manifest, componentName),
-		imageLink = checkAttr('imageLink', attributes, manifest, componentName),
 		imageAccept = checkAttr('imageAccept', attributes, manifest, componentName),
 		imageAllowedTypes = checkAttr('imageAllowedTypes', attributes, manifest, componentName),
-		imageBg = checkAttr('imageBg', attributes, manifest, componentName),
-		imageUsePlaceholder = checkAttr('imageUsePlaceholder', attributes, manifest, componentName),
+		imageFull = checkAttr('imageFull', attributes, manifest, componentName),
 
+		showImageUse = true,
 		showImageUrl = true,
 		showImageAlt = true,
-		showImageLink = true,
-		showImageBg = true,
+		showImageFull = true,
 	} = attributes;
 
+	const imageUrl = {
+		default: checkAttr('imageUrl', attributes, manifest, componentName),
+		desktop: checkAttr('imageUrlDesktop', attributes, manifest, componentName),
+		tablet: checkAttr('imageUrlTablet', attributes, manifest, componentName),
+		mobile: checkAttr('imageUrlMobile', attributes, manifest, componentName),
+	};
+
 	if (!imageShowControls) {
 		return null;
 	}
 
 	return (
 		<>
-
 			{label &&
 				<h3 className={'options-label'}>
 					{label}
 				</h3>
 			}
 
-			<ToggleControl
-				label={sprintf(__('Use %s', 'eightshift-frontend-libs'), label)}
-				checked={imageUse}
-				onChange={(value) => setAttributes({ [`${componentName}Use`]: value })}
-			/>
+			{showImageUse &&
+				<ToggleControl
+					label={sprintf(__('Use %s', 'eightshift-frontend-libs'), label)}
+					checked={imageUse}
+					onChange={(value) => setAttributes({ [`${componentName}Use`]: value })}
+				/>
+			}
 
 			{imageUse &&
 				<>
-					{(showImageUrl && imageUsePlaceholder && imageUrl === '') &&
-						<MediaPlaceholder
-							icon="format-image"
-							onSelect={(value) => {
-								setAttributes({
-									[`${componentName}Url`]: value.url,
-									[`${componentName}Alt`]: value.alt
-								});
-							}}
-							accept={imageAccept}
-							allowedTypes={imageAllowedTypes}
-						/>
+					{showImageUrl &&
+						<Responsive
+							label={
+								<>
+									<Icon icon={icons.link} />
+									{__('Image Url', 'eightshift-frontend-libs')}
```

Comment:
This could be moved to the toolbar to align with the recent Link and Button URL picker changes

Reply:
This is Image source so I have left on Image and remove the url from the string